content/english/python-basics/Loops/Break/_index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Break"
+draft: false
+weight: 1
+---
+
+You learned earlier that for loops and while loops keep iterating over a sequence until it reaches the end or until a condition is met.
+
+There is another way to stop running these loops before the end of the list or before a condition is met.
+
+You can do this with the break statement.  This lets you break out of the loop in either a for loop or while loop.  
+
+Here's an example with a while loop:
```

Comment:
Can you create an activity for testing your knowledge in break statements (and similar for other concepts)? See the C# curriculum as an example.

Reply:
Added a couple of activities